doc/requirements.txt

```diff
@@ -1,26 +1,722 @@
-# DOC: used to generate docs
+#
+# This file is autogenerated by pip-compile with Python 3.13
```

Comment:
what version are we using in CI? Should this match the same version used in CI?

Reply:
I'm not fluent in ruby, but AFAICS dependabot parses this line to match the python version.

Version parsing:
https://github.com/dependabot/dependabot-core/blob/0405a99f095c792385a61be4f0f11911b81a86e8/python/lib/dependabot/python/file_parser/python_requirement_parser.rb#L92
Matching:
https://github.com/dependabot/dependabot-core/blob/0405a99f095c792385a61be4f0f11911b81a86e8/python/lib/dependabot/python/language_version_manager.rb#L90

This kind of information lacks in the documentation it seems.

Reply:
the question is unrelated to dependabot , wondering if this would be a problem to generate hashes using one version and to be using a different version when installing the packages, i.e. there might be incompatibilities.

Reply:
I think we're good. The version is there to help dependabot generate requirements.txt files that are "consistent" (i.e. no unnecessary diffs). The actual hashes have nothing to do with it AFAICT and any pip 8+ version will be happy with the format of the file (https://pip.pypa.io/en/stable/topics/secure-installs/#hash-checking-mode)

Reply:
re-gen'd the requirements.txt with python 3.12 (Python version used in our docker image)